packages/generator-react-server/generators/app/templates/README.md

```diff
@@ -4,15 +4,62 @@ Very simple example project for `react-server` which only shows server rendering
 
 To start in development mode:
 
-```
+```shell
 npm start
 ```
 
 Then go to [localhost:3000](http://localhost:3000/). You will see a simple page that pre-renders and that is interactive on load. It also will include hot reloading of React components in their own file.
 
 If you want to optimize the client code at the expense of startup time, type `NODE_ENV=production npm start`. You can also use any of [the other arguments for react-server-cli](../../react-server-cli#setting-options-manually) after `--`. For example:
 
-```
+```shell
 # start in dev mode on port 4000
 npm start -- --port=4000
 ```
+
+# Developing using Docker and Docker Compose
+
+These steps assume you are familiar with docker and already have it installed. Some basics:
+
+1. Download [Docker Toolbox](https://www.docker.com/products/docker-toolbox) and install it.
+2. Start `docker quick start shell`
+3. Navigate to where you generated the project
+4. Add a configuration to set the `host` option to the ip given by `docker-machine ip`. An example configuration might be like:
+```json
+{
+  "port": "3000",
+  "env": {
+    "docker": {
+      "host": "Your ip from `docker-machine ip` here"
+    },
+    "staging": {
+      "port": "3000"
+    },
+    "production": {
+      "port": "80"
+    }
+  }
+}
```

Comment:
Just occurred to me that we should probably _generate_ this config.  https://github.com/redfin/react-server/issues/203